File: Modul-2/Soal-5/PRAK205-2410817210027-HafizPerdana.c
#include <stdio.h>
#include <math.h>

int main() {
    float sisi_a, sisi_b, alas, tinggi, keliling, luas;

    printf("");
    scanf("%f", &sisi_a);
    printf("");
    scanf("%f", &sisi_b);

    alas = sqrt(sisi_b * sisi_b - sisi_a * sisi_a);

    tinggi = sisi_a;

    keliling = sisi_a + sisi_b + alas;

    luas = 0.5 * alas * sisi_a;

    printf("\n");
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);

    return 0;
}

File: Modul-2/Soal-5/PRAK205-2410817210027-HafizPerdana.py
import math

sisi_a = float(input(""))
sisi_b = float(input(""))

alas = math.sqrt(sisi_b ** 2 - sisi_a ** 2)
tinggi = sisi_a
keliling = sisi_a + sisi_b + alas
luas = 0.5 * alas * sisi_a

print(f"\nAlas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")